cryptus-app/components/wallet_viewer/profile_wallet_viewer/collection_block/collection_nfts/collection_nfts.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from "react";
+import s from "./collection_nfts.module.scss";
+
+//external imports
+
+//internal imports
+import NFT_picture_block_in_wallet from "./nft_picture_block_in_wallet/nft_picture_block_in_wallet"
+import { nft } from "../../../../../lib/data_types"
+
+
+export default function Collection_NFTs(props: { nft: nft }) {
+  const { nft } = props;
+  
+
+  return (
+    <div className={s.container}>
+
+      <div className={s.block}>
+        <div className={s.grid2x2}>
+
+          <div>
+            <NFT_picture_block_in_wallet {...props} size={s.img_b}/>
+          </div>
+
+          <div>
+          <NFT_picture_block_in_wallet {...props} size={s.img_b}/>
+          </div>
+
+          <div>
+          <NFT_picture_block_in_wallet {...props} size={s.img_b}/>
+          </div>
+
+          <div>
+            <div className={s.grid2x2}>
+              <div>
+              <NFT_picture_block_in_wallet {...props} size={s.img_s}/>
+              </div>
+
+              <div>
+              <NFT_picture_block_in_wallet {...props} size={s.img_s}/>
+              </div>
+
+              <div>
+              <NFT_picture_block_in_wallet {...props} size={s.img_s}/>
+              </div>
+
+              <div>
+              <NFT_picture_block_in_wallet {...props} size={s.img_s}/>
+              </div>
+            </div>
+          </div>
+        </div>
+
+      </div>
+
+    </div>
+  );
+}
```

Comment:
use .map() and grid-areas. The code needs to work for a list of nfts, not just 1